learning/tour-of-beam/frontend/lib/pages/tour/widgets/content.dart

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
In this folder, most of the widgets are moved from `../screen.dart`, and their issues do not constitute a regression.